scripts/react-tester.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-env node */
+
+const { execSync } = require( 'child_process' );
+const { mkdirSync, rmdirSync, copyFileSync } = require( 'fs' );
+const { resolve: resolvePath } = require( 'path' );
+const satisfiesSemver = require( 'semver/functions/satisfies' );
+
+const PACKAGE_PATH = resolvePath( __dirname, '..' );
+const TESTS_PATH = resolvePath( PACKAGE_PATH, 'react-tests' );
+
+const packageInfo = require( '../package.json' );
+const availableVersions = getVersions();
+const semverRange = getReactVersion( packageInfo );
+const versionsToTest = getVersionsInRange( semverRange, availableVersions );
+
+try {
+	console.log( '--- Ultimate React Tester ---' );
```

Comment:
Yes it is! :smile: Or more like "Ultimate React CKEditor 4 Integration Tester" :stuck\_out\_tongue\_winking\_eye: